python/cugraph/tests/test_betweenness_centrality.py

```diff
@@ -37,7 +37,6 @@
 # =============================================================================
 DIRECTED_GRAPH_OPTIONS = [False, True]
 DEFAULT_EPSILON = 0.0001
-IMPLEMENTATION_OPTIONS = ['default', 'gunrock']
 
 TINY_DATASETS = ['../datasets/karate.csv']
```

Comment:
can we get these datasets from utils as well?

Reply:
I did want to creat yet another data test set.  This is slated for the next round of clean up and refactoring